### PR TITLE
Bento 2340- Dar component completed

### PR DESCRIPTION
**Pull request by Senait7-ux**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

**Comment by Senait7-ux** [comment]
https://github.com/CBIIT/bento-frontend/pull/774

**Review by iksheth — CHANGES_REQUESTED** [review]

File: packages/data-access-request/src/DataAccessRequest.js [hunk 1]
```diff
@@ -1,114 +1,75 @@
...
+import { Grid } from '@material-ui/core';
 import Button from '@material-ui/core/Button';
 import { useMutation } from '@apollo/client';
-import { useHistory, useLocation } from 'react-router-dom';
+import { useHistory} from 'react-router-dom';
 import CircularProgress from '@material-ui/core/CircularProgress';
-
-import { bentoHelpEmail } from '../../bento/userLoginData';
-import AlertMessage from '../../bento-core/AlertMessage';
+import AlertMessage from '../../bento-frontend/src/bento-core/AlertMessage';
```
[the remark this answers is not in the document]
Comment (Senait7-ux) [reply packages/data-access-request/src/DataAccessRequest.js:9]: Okay, will pass it as a prop.

**Comment by Senait7-ux** [comment]
https://github.com/CBIIT/bento-frontend/pull/774

File: packages/data-access-request/src/DataAccessRequest.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Senait7-ux) [reply packages/data-access-request/src/DataAccessRequest.js]: https://github.com/CBIIT/bento-frontend/pull/774

**Review by iksheth — APPROVED** [review]

**Comment by iksheth** [comment]
Looks good to me. Merging this PR & Deploying to DEV.

**Pull request merged**